app/src/main/res/values/strings.xml

```diff
@@ -632,6 +639,12 @@
     <string name="settings_theme_heading">Appearance</string>
     <string name="settings_theme_desc">Dark/light theme</string>
 
+    <string name="settings_notification_heading">Choose Notification Action</string>
+    <string name="settings_notification_desc">Current action button for notification : %1$s</string>
+    <string name="settings_notification_desc1">Nothing</string>
+    <string name="settings_notification_desc2">STOP</string>
+    <string name="settings_notification_desc3">DNS &amp; DNS+Firewall</string>
```

Comment:
1. Stop (default): Shows one button, tapping on which stops RethinkDNS
2. DNS / DNS + Firewall: Shows two buttons to quickly switch between the battery-saver DNS mode and the regular DNS + Firewall mode.
3. Empty: Shows no button.

Btw, when the modes are switched (`#2`), does the notification text change? That is,

`Protected! Connected to Secure DNS + Firewall.` to `Protected! Connected to Fast and Secure DNS` and vice-versa? If not, we should do so, otherwise, there's no visual feedback for the user on what happened. Showing a `toast` is another way out, if it could be shown even when RethinkDNS isn't in the foreground.

Reply:
Yes when the modes are modified, then the text in the notification will change. 
For DNS - Protected! Connected to Fast and Secure DNS.
For DNS + Firewall - Protected! Connected to Secure DNS + Firewall.